satpy/scene.py

```diff
@@ -101,11 +101,15 @@ def __init__(self, filenames=None, reader=None, filter_parameters=None,
         will be searched for a Reader that can support the provided files. This
         can take a considerable amount of time so it is recommended that
         ``reader`` always be provided. Note without ``filenames`` the Scene is
-        created with no Readers available requiring Datasets to be added
-        manually::
+        created with no Readers available.  When a Scene is created with no Readers,
```

Comment:
For consistency:

```suggestion
        created with no Readers available. When a Scene is created with no Readers,
```

As a former two-spacer myself, apparently it is the old not-modern way to write things. I'm still doing my oxford commas though! :angry: